linker/shared/src/main/scala/org/scalajs/linker/standard/ModuleSet.scala

```diff
@@ -0,0 +1,45 @@
+/*
+ * Scala.js (https://www.scala-js.org/)
+ *
+ * Copyright EPFL.
+ *
+ * Licensed under Apache License 2.0
+ * (https://www.apache.org/licenses/LICENSE-2.0).
+ *
+ * See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.
+ */
+
+package org.scalajs.linker.standard
+
+import org.scalajs.ir.Names.ClassName
+import org.scalajs.ir.Trees.TopLevelExportDef
+
+import org.scalajs.linker.interface.ModuleInitializer
+import org.scalajs.linker.interface.ModuleSpec.ModuleID
+
+final class ModuleSet private[linker] (
```

Comment:
An alternative is to evolve the `LinkingUnit` into this class instead. What is unclear is how the `Refiner` would deal with a non-single module `LinkingUnit`.

Reply:
No, I think what you have now is better. It's better if front-end phases (the optimizer, but also future custom front-end phases) don't have to worry about modules.